xyz.judro.mines.yml

```diff
@@ -0,0 +1,22 @@
+app-id: xyz.judro.mines
```

Comment:
`judro.xyz` is NXDOMAIN. so it's likely the app-id doesn't meet the requirement of "under the control of the developers"